code/modules/vending/vending.dm

```diff
@@ -1174,6 +1174,8 @@
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/antihistamine, 2, cost=PAY_DOCTORATE/5)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/ampoule/smelling_salts, 2, cost=PAY_DOCTORATE/10)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/pill/salicylic_acid, 10, cost=PAY_DOCTORATE/10)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/mask/surgical, 10, cost=PAY_DOCTORATE/10)
+		product_list += new/datum/data/vending_product(/obj/item/clothing/gloves/latex, 10, cost=PAY_DOCTORATE/10)
```

Comment:
Maybe reduce to PAY_TRADESMAN/10 or PAY_UNTRAINED/10?

Reply:
I mean...everything else in this vendor is based on PAY_DOCTORATE, so it would feel a little strange...

Reply:
i'm going to do the math real quick:
DOCTORATE base: 480/10 = 48 //the current price, half of a staff assistant's starting credits (not counting the 100 in the ATM)
TRADESMAN base: 300/10 = 30 //actually pretty reasonable, but feels weird being tied to the trade budget
UNTRAINED base: 120/10 = 12 //too cheap and will be bought out immediately. only real countermeasure is quartermaster restocking it, which is fine in planning, but in practice, there ususally isn't enough communication to get quartermasters to stock vendors quickly enough for masks to make much of a difference for a pathogen. arguably, they should be alert when there's a pathogen already because they hold the CDC access, but that's also not going to be expected of them on nonrp.

possible other modifiers:
DOCTORATE/20 Base: 480/20 = 24 //a fourth of a staff assistant's starting cash
DOCTORATE/15 Base: 480/15 = 32 //about the same as trade budget

so, if you reeeally feel like the price should  be reduced, i feel like DOCTORATE/15 is the best middleground option

Reply:
Requested change more on the basis of: if we were to change the pay tier values, I feel it would make more sense for the cost to be based around the payscale of untrained/tradesman as opposed to doctorate level pay

Reply:
that would make sense...on that topic though...i feel like all of the prices in the minimed should scale off of untrained level pay as medical doctors arent going to be buying from them...

Reply:
i'll look into this a little more later. I'm a little wrapped up in the beeeeeg cake patch ( ^ - ^ )

Reply:
alrighty, so i did some testing and poking around on gooncord and people seemed to like these numbers the best:

original:
![x770pFnxED](https://user-images.githubusercontent.com/12144569/105560027-2f7a8680-5cbf-11eb-89eb-3111cc56563c.png)

new:
![iHAr2QyTBH](https://user-images.githubusercontent.com/12144569/105560034-35706780-5cbf-11eb-9c44-f65750199879.png)